### ISSUE TITLE
Real Math Italic?

### ISSUE THREAD

**Issue by RuixiZhang42** [firamath/firamath#38]
This is probably asking too much from an open-source project, but I&nbsp;think it is worth mentioning nonetheless.

Professional math fonts seem to set their math italic slightly different from the corresponding text italic. To quote from the designer of the (expensive) [Minion Math](http://www.typoma.com/en/fonts.html):
> … with a real Math Italic alphabet (wider than Text Italic, and with special kerning).

For an extended read, please refer to [Fonts for Mathematics (PDF)](http://www.typoma.com/publ/20041002-atypi.pdf), [Minion Math — The Design of a New Math Font Family (YouTube)](https://www.youtube.com/watch?v=Xw7oBf8EmoM) and [Is there a perfect maths font?](http://chalkdustmagazine.com/blog/is-there-a-perfect-maths-font/).

## Examples

I&nbsp;have collected 3 examples reflecting the above claim. Unfortunately, the TeX Gyre collection does *not* follow this design guide AFAICT. In the following pictures, text italics are in red, math italics are in blue.

> ![cm](https://user-images.githubusercontent.com/39968857/58290007-2ddc1b00-7d86-11e9-8b77-4ade5c0cba2e.png)<br>Computer Modern Italic versus Computer Modern Math Italic

> ![times](https://user-images.githubusercontent.com/39968857/58290028-3f252780-7d86-11e9-93eb-b0ac56a02054.png)<br>Times New Roman Italic versus MathTime Professional&nbsp;2

> ![stix2](https://user-images.githubusercontent.com/39968857/58290057-6aa81200-7d86-11e9-9788-7120180b7dde.png)<br>STIX2 Text Italic versus STIX2 Math Italic

**Comment by RuixiZhang42** [comment]
With the release of [Source Han Mono](https://github.com/adobe-fonts/source-han-mono), I&nbsp;came across with the following description in the official ReadMe document in the *Anisotropic Techniques* section:

> Anisotropic techniques, which can preserve vertical stem weights, were used to horizontally-expand the half-width katakana masters to 667 units, and the same techniques were applied to the masters for the hangul letters and syllables, to make them 1000 units. A small number of additional glyphs required similar treatment.

This was necessary as the half-width katakana were 500 units wide and the hangul were 920 units wide.

Is it possible to apply such technique to Fira Math? I&nbsp;mean, using anisotropic techniques we could preserve the *weight* of the glyph while expanding its width. This would be perfect for creating real math italic as well as [optical sizes](https://github.com/firamath/firamath/issues/23).

---

Dr.&nbsp;Ken Lunde explained this technique in much more detail in his blog post [*Source Han Mono Version 1.000 Technical Nuggets*](http://blogs.adobe.com/CCJKType/2019/05/source-han-mono-v1000.html). In particular, the horizontal expansions were 133.4% for the katakana and 108.7% for the hangul. Considered that math italics usually are just 105% of the text italics, I&nbsp;really think this can work.

**Comment by stone-zeng** [comment]
I guess it's much closer to FontForge's [Stylistic Transformations](https://fontforge.github.io/Styles.html). There is such things for Change Weight, Italic, Condense/Extend, etc, but I haven't checked them carefully.

Anyway, this issue (and the optical size) will not be considered recently, until I have added enough glyphs and correct kerning, which is much more important at present.

**Comment by stone-zeng** [comment]
Note that the anisotropic techniques used in Source Han Mono are not perfect, for example the stem width is not thick enough. This seems to be one of the reason for the [1.001 update](http://blogs.adobe.com/CCJKType/2019/05/source-han-mono-v1001.html).

**Comment by RuixiZhang42** [comment]
> Note that the anisotropic techniques used in Source Han Mono are not perfect[…]

Yes, both the *proportion of the character forms* and the *weight* of the expanded half-width katakana were mismatched and they looked ugly in v1.000. But this was the result of a **horizontal expansion of 133.4%**(!!!) as I commented earlier. The redesign of these expanded katakana was much expected.

Note that the 920-unit wide hangul letters and syllables transformations remain **unchanged**, since their horizontal expansion is only 108.7%. The designer of Minion Math suggests a range of 105%–110%, and I&nbsp;don’t think anything would look good beyond this range before manual redesign.
